mifosng-android/src/main/res/layout/fragment_generate_collection_sheet.xml

```diff
@@ -2,158 +2,183 @@
   ~ This project is licensed under the open source MPL V2.
   ~ See https://github.com/openMF/android-client/blob/master/LICENSE.md
   -->
-<ViewFlipper xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@therajanmaurya Could you please tell what would be the benefit of using `NestedScrollView` here instead of `ScrollView` as there is only single vertical scrolling layout?